specifyweb/frontend/js_src/lib/components/WebLinks/index.tsx

```diff
@@ -59,13 +52,9 @@ export function WebLinkField({
   const [builtUrl, setUrl] = React.useState<
     RA<string | { readonly prompt: string }> | undefined
```

Comment:
```suggestion
  RA<string | undefined
```

can you do a global case-insenstive search on "prompt" just in case we missed any other places?